os/pm/pm_idle.c

```diff
@@ -111,11 +109,8 @@ void pm_idle(void)
 			}
 		}
 #endif
-#ifdef CONFIG_PM_TICKSUPPRESS
-	up_pm_board_sleep(pm_wakehandler);
-#else
-	up_pm_board_sleep(NULL);
-#endif
+		up_pm_board_sleep(pm_wakehandler);
+		stime = clock_systimer();
```

Comment:
what is the use of now and stime , it will be always now > stime . 
what is the use of if (now > stime) condition?

Reply:
In one tick `pm_idle` can run multiple times, to avoid this we have added this condition